airflow/jobs.py

```diff
@@ -721,6 +721,25 @@ def create_dag_run(self, dag, session=None):
             if dag.schedule_interval == '@once' and last_scheduled_run:
                 return None
 
+            # don't do backfill for dag's that don't have dag.backfill = True
+            if not dag.backfill:
+                # The logic is that we move start_date up until
+                # one period before, so that datetime.now() is AFTER
+                # the period end, and the job can be created...
```

Comment:
The way the scheduler is written, I'm not sure there is a clean way to do this otherwise. @bolkedebruin and I talked this over before I started on this and we landed on this as the only clean way to do this. In the same way that dependencies have been refactored recently, scheduler needs a refactor. However, I think that is best left to 2.0 and this is a clean way to keep the scheduler from running backfills with the least code change (less chance of bugs).